src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,547 @@
+        {blueprintName && (
+          <>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Image name
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {blueprintName}
+            </TextListItem>
+          </>
+        )}
+        {blueprintDescription && (
+          <>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Description
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {blueprintDescription}
+            </TextListItem>
+          </>
+        )}
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const OscapList = () => {
+  const oscapProfile = useAppSelector((state) => selectProfile(state));
+  const release = useAppSelector((state) => selectDistribution(state));
+  const { data } = useGetOscapCustomizationsQuery(
+    {
+      distribution: release,
+      // @ts-ignore if oscapProfile is undefined the query is going to get skipped, so it's safe here to ignore the linter here
+      profile: oscapProfile,
+    },
+    {
+      skip: !oscapProfile,
+    }
+  );
+
+  const enabledServices = data?.services?.enabled;
+  const enabledServicesDisplayString = enabledServices?.reduce(
+    (accumulator, currentValue) => accumulator + ' ' + currentValue,
+    ''
+  );
+  const disableServices = data?.services?.disabled;
+  const disableServicesDisplayString = disableServices?.reduce(
+    (accumulator, currentValue) => accumulator + ' ' + currentValue,
+    ''
+  );
+  return (
+    <TextContent>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Profile name:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {data?.openscap?.profile_name}
+        </TextListItem>
+      </TextList>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Profile description:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {data?.openscap?.profile_description}
+        </TextListItem>
+      </TextList>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Reference ID:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {oscapProfile}
+        </TextListItem>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
```

Comment:
Do we want to render an empty code block when there are no kernel arguments / services? I guess we maybe do, right? Just to have a complete overview of what's included in the profile?

Reply:
this is a very good question, I thought about that and maybe its intersting the user to see the both case - when there is existning data for kernel and services and also when there is no for a specific pofile

Reply:
Agree, this way the data may render as empty, but the overview doesn't feel incomplete... maybe we could render "None" instead of the empty code block, but that might be something we might want to ask during the UX sync

Reply:
I spoke with Kelsea and she said in case we dont have any value, keep it empy box